server/src/main/java/com/lunchtime/models/Restaurant.java

```diff
@@ -0,0 +1,89 @@
+package com.lunchtime.models;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import java.time.Instant;
+
+@Entity
+@Where(clause = "is_deleted = false or is_deleted is NULL")
+@Setter
+@Getter
+public class Restaurant {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @NotBlank
+    @Column(name = "name")
+    private String name;
+
+    @NotBlank
+    @Email
+    @Column(name = "email")
+    private String email;
+
+    @NotBlank
+    @Column(name = "text_address")
+    private String textAddress;
+
+    @Column(name = "website")
+    private String website;
+
+    @NotBlank
+    @Column(name = "description")
+    private String description;
+
+    @NotBlank
+    @Column(name = "working_time")
+    private String workingTime;
+
+    @Column(name = "is_deleted")
+    private Boolean isDeleted;
+
+    @Column(name = "menu_id")
+    private Long menuId;
+
+    @Column(name = "owner_id")
+    private Long ownerId;
+
+    @Column(name = "tables")
+    private Integer tables;
+
+    @Column(name = "longitude")
+    private Float longitude;
+
+    @Column(name = "latitude")
+    private Float latitude;
+
+    @Column(name = "created_at")
+    @CreationTimestamp
+    private Instant createdAt;
+
+    @Column(name = "created_by")
+    private Long createdBy;
+
+    @Column(name = "modify_at")
+    @UpdateTimestamp
+    private Instant modifyAt;
+
+    @Column(name = "modify_by")
+    private Long modifyBy;
+
+    public Boolean getIsDeleted() {
```

Comment:
Due to JavaBean Specifications (**8.3.2 Boolean properties**), we should use:
public boolean (or Boolean) **is**Deleted<PropertyName>() instead of **getIs**Deleted<PropertyName>()

Also we can use a ternary operator for its readability

Reply:
Unfortunately I didn't find this rule, but I think you're right

Reply:
Thank you for reliance! :)

**[Here ](https://download.oracle.com/otndocs/jcp/7224-javabeans-1.01-fr-spec-oth-JSpec/)** you can download it and see 8.3.2 paragraph. I also attached an image from this file, supporting this recommendation.

![t-bp](https://user-images.githubusercontent.com/50665431/76399332-1d357980-6387-11ea-958a-df3c4ade1f98.png)

Reply:
I am completely confused)
Should I rename property `private Boolean isDeleted` to `private Boolean deleted`?
Because pattern `get<PropertyName>` where `PropertyName` is `isDeleted` was realized in previous code iteration

Reply:
If I understand you correctly (line 19), I would keep property `private Boolean isDeleted` and remane `getIsDeleted()` to `isDeleted()`. And we will have

```
private Boolean isDeleted
public Boolean isDeleted()
```

Reply:
Fixed